packages/data/src/world/oot/overworld.yml

```diff
@@ -2080,12 +2080,28 @@
     "Lake Hylia Gossip Near River": "true"
     "Lake Hylia Gossip Southeast Lake": "true"
     "Lake Hylia Gossip Southwest Lake": "true"
+"Lake Hylia Near Shortcut":
+  region: LAKE_HYLIA
+  age_change: false
+  exits:
+    "Lake Hylia": "is_adult || can_swim"
+    "Zora Domain Shortcut": "is_adult || can_swim"
```

Comment:
We can remove this exit, since the entry from Lake Hylia region guarantees access and any methods to make sure you stay underwater will also get you to Lake Hylia and then back in.